app/code/Magento/Bundle/view/adminhtml/templates/product/composite/fieldset/options/type/checkbox.phtml

```diff
@@ -3,16 +3,24 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
+
+use Magento\Bundle\Block\Adminhtml\Catalog\Product\Composite\Fieldset\Options\Type\Checkbox;
+use Magento\Framework\Escaper;
+
+/** @var Escaper $escaper */
+/** @var Checkbox $block */
+$_option = $block->getOption();
```

Comment:
Due to file update, could you please update variable names and remove underscore?

Reply:
Is this a blocking request for change? I'd rather not refactor more than I have to. While I agree that removing leading underscores from variable names is preferred, not doing it as part of this change doesn't make anything worse than it already is.